platform/utils.go

```diff
@@ -0,0 +1,56 @@
+package platform
```

Comment:
These functions should eventually go somewhere else... but I think their new homes will be created when we do additional cleanup as part of https://github.com/buildpacks/rfcs/pull/217